<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect to RiskSpectrum DB model</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="loadFromMemory">
        <property name="text">
         <string>Select model:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1,0">
        <item>
         <widget class="QComboBox" name="models"/>
        </item>
        <item>
         <widget class="QPushButton" name="updateModelList">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>Updating the list of active models</string>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="detachButton">
          <property name="whatsThis">
           <string>Unload Model</string>
          </property>
          <property name="text">
           <string>Detach</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="loadFromFile">
        <property name="text">
         <string>Open from file:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,0,0">
        <item>
         <widget class="QLineEdit" name="modelFilePath">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openFileModel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Open file dialog</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="attachButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>sdf</string>
          </property>
          <property name="whatsThis">
           <string>Load model into SQL server</string>
          </property>
          <property name="text">
           <string>Attach</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Server connection parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,10">
      <item row="1" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serverName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>.\RISKSPEC_PSA2012</string>
        </property>
        <item>
         <property name="text">
          <string>.\RISKSPEC_PSA2012</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="pwd">
        <property name="text">
         <string>82sbDiF%5_2&amp;33d%hvTP!4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Server name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="uid">
        <property name="text">
         <string>sa</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFromMemory</sender>
   <signal>toggled(bool)</signal>
   <receiver>models</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFromFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>modelFilePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFromFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>openFileModel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFromMemory</sender>
   <signal>toggled(bool)</signal>
   <receiver>updateModelList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFromMemory</sender>
   <signal>toggled(bool)</signal>
   <receiver>detachButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadFromFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>attachButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
